packages/directories/directories.0.6/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis:
+  "An OCaml library that provides configuration, cache and data paths (and more!) following the suitable conventions on Linux, macOS and Windows"
+description:
+  "directories is an OCaml library that provides configuration, cache and data paths (and more!) following the suitable conventions on Linux, macOS and Windows. It is inspired by similar libraries for other languages such as directories-jvm. The following conventions are used: XDG Base Directory Specification and xdg-user-dirs on Linux, Known Folders on Windows, Standard Directories on macOS."
+maintainer: ["OCamlPro <contact@ocamlpro.com>"]
+authors: ["OCamlPro <contact@ocamlpro.com>"]
+license: "ISC"
+homepage: "https://github.com/ocamlpro/directories"
+bug-reports: "https://github.com/ocamlpro/directories/issues"
+depends: [
+  "dune" {>= "2.1"}
+  "ocaml" {>= "4.14.0"}
+  "ctypes" {>= "0.17.1" & (os = "win32" | os = "cygwin")}
+  "fpath"
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
I missed this in the last review, but it is being called out by the failing linting check:

```suggestion
  ["dune" "subst"] {dev}
```

See https://github.com/ocaml/dune/pull/3647 for context if needed.